.github/workflows/aks-deploy.yaml

```diff
@@ -186,6 +187,9 @@ jobs:
             }
           }
           
+          $release = "$appName-${{ github.sha }}".Substring(0,53)
```

Comment:
On the Substring call here is there a more reliable way to assign this?  I don't know what I don't know here but if that $release var gets viewed in K8s or docker, I wonder if we could make it something like:
$appName-$appEnvironment-$appVersion-(last 10 characters of the github sha) or something like that, maybe?  Not a show stopper just thought I'd ask!

Reply:
Well previously, this was set by a whole other step entirely. I just moved it to this step to fix a silly linting error I was getting.

Here is where it used to be before: https://github.com/Andrews-McMeel-Universal/reusable_workflows/pull/10/files#diff-377d534d3f697f70d4e4ae1913ade4d3d1ef69befae238c4af735a7a0a5558faL304-L311

As far as changing it to simplify the image tag or doing something similar goes, it wouldn't make too much a difference if we name it differently as we really don't touch those images once they've been added to the registry unless I'm forgetting about something.

Maybe once we figure out a good way to use existing images when promoting to the different environments, we could revisit this though.